bedrock/base/templates/includes/canonical-url.html

```diff
@@ -10,7 +10,7 @@
         {% if code == 'en-US' -%}
         <link rel="alternate" hreflang="en" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English">
         <link rel="alternate" hreflang="en-CA" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English (Canada)">
-        {% else -%}
+        {% elif code not in ['ast', 'cak', 'dsb', 'hsb', 'kab', 'lij', 'son'] -%}{#- Bug 1364470: drop locales not supported by Google -#}
```

Comment:
Yeah, so I've just updated the template to simply check the length of locale codes.